packages/slice-machine/tests/server/slices/delete.test.ts

```diff
@@ -152,6 +152,16 @@ const CustomTypeModelWithoutSlice = {
   },
 };
 
+const emptyIndexFile = `// Code generated by Slice Machine. DO NOT EDIT.
```

Comment:
OPT:

Could this be exported to reinforce the tests ?

Reply:
Not really sure what you mean by this?  Is it so others can use it?

Reply:
I meant if the original text change, this could make the test fail maybe ?
So by importing the text itself, maybe we could avoid this kind of situation.

If that's not easily do-able forget about it 😄 , i just thought of it as a way to improve the tests